resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -147,6 +147,15 @@ private[spark] object Config extends Logging {
   val CLIENT_CERT_FILE_CONF_SUFFIX = "clientCertFile"
   val CA_CERT_FILE_CONF_SUFFIX = "caCertFile"
 
+
+  val KUBERNETES_CLIENT_OAUTH_TOKEN_PROVIDER =
+    ConfigBuilder("spark.kubernetes.client.oauth.token.provider.class")
+      .doc("A class that implements fabric's OAuthTokenProvider interface to " +
```

Comment:
Could you remove `fabric's`?

Reply:
Thanks for the review! Removed it.
But now I am not sure if it is obvious which interface (it's fabric8's `io.fabric8.kubernetes.client.OAuthTokenProvider`) needs to be implemented and passed for this to work.

Maybe a somewhat more verbose description would be better? Something like
> A class that implements OAuthTokenProvider interface to provide a OAuth refresh mechanism for long running jobs. The class must be in a driver's classpath.

Reply:
Thanks! Updated.